### PR TITLE
F5 rewrite

### PR DESCRIPTION
**Pull request by bradleycamacho**

### CODE REVIEW

**Comment by akristen** [comment]
Looks like the build is still broken. I'll see if I can fix it or get it to work on a local build, then leave a general comment here if I can.

File: src/install/f5/linux/install-tarball.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    To install an integration from a tarball:
+
+    1. Download the packaged integration file from the [tarball repository](https://download.newrelic.com/infrastructure_agent/binaries/linux/amd64/).
+    2. Unpack the tarball file according to our [integration file structure and placement rules](/docs/integrations/integrations-sdk/getting-started/integration-file-structure-activation), so the agent is able to find the definitions, configurations, and executables of the integration.
+    3. Place the binary that contains the definition file inside `newrelic-integrations` or `custom-integrations` in the [agent directory](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux#agent-directory).
+    4. Place the integration's config file under the [plugin directory](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux#configure-plugin).
+    5. Follow the JMX tool instructions below.
```
Comment (nbaenam) [inline src/install/f5/linux/install-tarball.mdx:18]: I think we decided to change step 5. Check this: https://docs.newrelic.com/install/ibm-mq/?deployment=tar. Let me know if I should change it.

File: src/install/f5/linux/install-yum.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (nbaenam) [inline src/install/f5/linux/install-yum.mdx:1]: I have the same comment that I've added for install-apt.mdx

File: src/install/f5/linux/install-zypper.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (nbaenam) [inline src/install/f5/linux/install-zypper.mdx:1]: I have the same comment that I've added for install-apt.mdx

File: src/install/f5/whatsNext.mdx [hunk 4]
```diff
@@ -0,0 +1,2055 @@
...
+  </tbody>
+</table>
+
+You can define these setting values in different ways, depending on your preference and need:
+
+* Add the value directly in the config file. This is the most common way.
+* Replace the values from environment variables using the `{{}}` notation. This requires infrastructure agent v1.14.0 or higher. Read more [on using passthrough](/docs/infrastructure/install-infrastructure-agent/configuration/configure-infrastructure-agent/#passthrough) or see the [environment variables replacement example](#envvar-replacement).
+* Use secrets management to protect sensitive information, such as passwords, that would be exposed in plain text in the configuration file. For more information, read more about using [secrets management](/docs/integrations/host-integrations/installation/secrets-management).
+
+## Labels/custom attributes [#labels]
```
Comment (nbaenam) [inline src/install/f5/whatsNext.mdx:298]: I believe that we should avoid the use of `/`

File: src/install/f5/whatsNext.mdx [hunk 5]
```diff
@@ -0,0 +1,2055 @@
...
+## Example configurations [#examples]
+
+<CollapserGroup>
+  <Collapser
+    id="basic-config"
+    title="Basic configuration"
+  >
+    This is the basic configuration used to collect metrics and inventory from your localhost:
+
+    ```
```
Comment (nbaenam) [inline src/install/f5/whatsNext.mdx:319]: I'd add yml here.

File: src/install/f5/whatsNext.mdx [hunk 6]
```diff
@@ -0,0 +1,2055 @@
...
+* Replace the values from environment variables using the `{{}}` notation. This requires infrastructure agent v1.14.0 or higher. Read more [on using passthrough](/docs/infrastructure/install-infrastructure-agent/configuration/configure-infrastructure-agent/#passthrough) or see the [environment variables replacement example](#envvar-replacement).
+* Use secrets management to protect sensitive information, such as passwords, that would be exposed in plain text in the configuration file. For more information, read more about using [secrets management](/docs/integrations/host-integrations/installation/secrets-management).
+
+## Labels/custom attributes [#labels]
+
+You can also decorate your metrics using labels. Labels allow you to add key/value pair attributes to your metrics, which you can then use to query, filter, or group your metrics.
+
+Our default sample config file includes examples of labels. You can remove, modify, or add new ones of your choice.
+
+```
```
Comment (nbaenam) [inline src/install/f5/whatsNext.mdx:304]: I'd add yml here.

File: src/install/f5/whatsNext.mdx [hunk 7]
```diff
@@ -0,0 +1,2055 @@
...
+    ```
+  </Collapser>
+
+  <Collapser
+    id="basic-intervals"
+    title="Basic configuration with different metric/inventory intervals"
+  >
+    This configuration collects metrics every 15 seconds and inventory every 60 seconds:
+
+    ```
```
Comment (nbaenam) [inline src/install/f5/whatsNext.mdx:340]: I'd add yml here and in the following chunk of code.

File: src/install/f5/whatsNext.mdx [hunk 8]
```diff
@@ -0,0 +1,2055 @@
...
+        interval: 15s
+        labels:
+          environment: production
+        inventory_source: config/f5
+    ```
+  </Collapser>
+
+  <Collapser
+    id="basic-intervals"
+    title="Basic configuration with different metric/inventory intervals"
```
Comment (nbaenam) [inline src/install/f5/whatsNext.mdx:336]: I believe that we should avoid the use of /

File: src/install/f5/windows/install-windows.mdx [hunk 9]
```diff
@@ -0,0 +1,29 @@
+---
+componentType: default
+headingText: Configure the F5 integration
+---
+
+1. In the Integrations directory, `C:\Program Files\New Relic\newrelic-infra\integrations.d\`, create a copy of the sample configuration file by running:
+
+```shell
+copy F53-config.yml.sample F5-config.yml
```
Comment (nbaenam) [inline src/install/f5/windows/install-windows.mdx:9]: I think there is a typo here: `F53-config.yml.sample`

**Review by nbaenam — COMMENTED** [review]
Hey @bradleycamacho, I added some comments. Let me know if you have questions.

**Comment by bradleycamacho** [comment]
Moved to https://github.com/newrelic/docs-website/pull/16487

**Pull request closed without merging**